src/coffee/directives/api/options/builders/common-options-builder.coffee

```diff
@@ -3,8 +3,6 @@ angular.module('uiGmapgoogle-maps.directives.api.options.builders')
 [ 'uiGmapBaseObject', 'uiGmapLogger', 'uiGmapModelKey', (BaseObject, $log, ModelKey) ->
 
   class CommonOptionsBuilder extends ModelKey
-    constructor: ->
-      @hasModel = _(@scope).chain().keys().contains('model').value()
```

Comment:
This was causing issues in some async issues. Basically some race when scope.model didb't but now exists.